src/Components/Clock/Clock.tsx

```diff
@@ -0,0 +1,54 @@
+import { Component, ReactNode } from 'react';
+
+type State = {
+  today: Date,
+};
+
+type Props = {
+  clockName: string,
+};
+export class Clock extends Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  clockId: number | null = null;
+
+  componentDidMount() {
+    this.clockId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
```

Comment:
will be better to create separate function for transforming date, because for now you repeated the same code twice